pkg/validate/print.go

```diff
@@ -1,32 +1,49 @@
 package validate
 
 import (
-	"fmt"
-	"strings"
-
+	"github.com/alexeyco/simpletable"
 	"github.com/mt-sre/addon-metadata-operator/pkg/utils"
 )
 
 var (
-	equalLines = strings.Repeat("=", 6)
-
-	success = utils.Green("Success")
-	failed  = utils.Red("Failed")
-	err     = utils.IntenselyBoldRed("Error")
+	statusSuccess = utils.Green("Success")
+	statusFailed  = utils.Red("Failed")
+	statusError   = utils.IntenselyBoldRed("Error")
 )
 
-func printMetaHeading() {
-	fmt.Printf("\n%sRUNNING METADATA VALIDATORS%s\n\n", equalLines, equalLines)
+func getTableHeaders() *simpletable.Header {
+	return &simpletable.Header{
+		Cells: []*simpletable.Cell{
+			{Align: simpletable.AlignCenter, Text: "STATUS"},
+			{Align: simpletable.AlignCenter, Text: "CODE"},
+			{Align: simpletable.AlignCenter, Text: "NAME"},
+			{Align: simpletable.AlignCenter, Text: "DESCRIPTION"},
+			{Align: simpletable.AlignCenter, Text: "FAILURE MESSAGE"},
+		},
+	}
+}
+
+func newSuccessTableRow(v utils.Validator) []*simpletable.Cell {
+	return newTableRow(v, statusSuccess, "")
 }
 
-func printSuccessMessage(msg string) {
-	fmt.Printf("\r%s\t\t%s", msg, success)
+func newFailedTableRow(v utils.Validator, failureMsg string) []*simpletable.Cell {
+	return newTableRow(v, statusFailed, failureMsg)
 }
 
-func printFailureMessage(msg string) {
-	fmt.Printf("\r%s\t\t%s", msg, failed)
+func newErrorTableRow(v utils.Validator, err error) []*simpletable.Cell {
+	return newTableRow(v, statusError, err.Error())
 }
 
-func printErrorMessage(msg string) {
-	fmt.Printf("\r%s\t\t%s", msg, err)
+func newTableRow(v utils.Validator, status, failureMsg string) []*simpletable.Cell {
```

Comment:
Any specific reason why these functions are intended to return pointer `[]*simpletable.Cell` and not value `[]simpletable.Cell`?

Reply:
Because on the other side, in `validate.go`, you have `t.Header` + `t.Body.Cells` which require pointers